scripts/apps/contacts/Contacts.d.ts

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
Use \*.ts/\*.tsx files instead of *d.ts when you can. *d.ts ones don't get type-checked properly. It might be an issue with our setup, but it's easier to change the extension than investigate it.